src/utils.py

```diff
@@ -1,7 +1,48 @@
 import base64
+import pandas as pd
 
 def get_table_download_link_csv(df):
     csv = df.to_csv().encode()
     b64 = base64.b64encode(csv).decode()
     href = f'<a href="data:file/csv;base64,{b64}" download="ot_metrics_data.csv" target="_blank">Download data</a>'
-    return href
\ No newline at end of file
+    return href
+
+def compare_evidence(
+    evidence: pd.DataFrame,
+    latest_run: str,
+    previous_run: str
+) -> pd.DataFrame:
+    evidence["Δ in number of evidence strings"] = evidence[f"Nr of evidence strings in {latest_run.split('-')[0]}"].sub(evidence[f"Nr of evidence strings in {previous_run.split('-')[0]}"], axis=0, fill_value=0)
+    evidence["Δ in number of invalid evidence strings"] = evidence[f"Nr of invalid evidence strings in {latest_run.split('-')[0]}"].sub(evidence[f"Nr of invalid evidence strings in {previous_run.split('-')[0]}"], axis=0, fill_value=0)
+    evidence["Δ in number of evidence strings dropped due to duplication"] = evidence[f"Nr of evidence strings dropped due to duplication in {latest_run.split('-')[0]}"].sub(evidence[f"Nr of evidence strings dropped due to duplication in {previous_run.split('-')[0]}"], axis=0, fill_value=0)
+    evidence["Δ in number of evidence strings dropped due to unresolved target"] = evidence[f"Nr of evidence strings dropped due to unresolved target in {latest_run.split('-')[0]}"].sub(evidence[f"Nr of evidence strings dropped due to unresolved target in {previous_run.split('-')[0]}"], axis=0, fill_value=0)
+    evidence["Δ in number of evidence strings dropped due to unresolved disease"] = evidence[f"Nr of evidence strings dropped due to unresolved disease in {latest_run.split('-')[0]}"].sub(evidence[f"Nr of evidence strings dropped due to unresolved disease in {previous_run.split('-')[0]}"], axis=0, fill_value=0)
```

Comment:
I suggest rewrapping these strings to <120 characters as per our version of PEP8. This could also improve readability

Reply:
Also a few such occasions in other `compare_*` functions below

Reply:
And just to give an example of how I think this could be re-wrapped:

```python
    evidence["Δ in number of evidence strings dropped due to unresolved disease"] = (
        evidence[f"Nr of evidence strings dropped due to unresolved disease in {latest_run.split('-')[0]}"]
        .sub(
            evidence[f"Nr of evidence strings dropped due to unresolved disease in {previous_run.split('-')[0]}"],
            axis=0, fill_value=0
        )
    )
```

Reply:
And also also, it seems to me that adding all of the deltas could be simplified by adding a function like:

```python
def add_delta(evidence, metric, previous_run, latest_run):
    evidence[f"Δ in number of {metric}"] = (
        evidence[f"Nr of {metric} in {latest_run.split('-')[0]}"]
        .sub(
            evidence[f"Nr of {metric} in {previous_run.split('-')[0]}"],
            axis=0, fill_value=0
        )
    )
```

And then the above example could just be rewritten as
```python
    add_delta(evidence, 'evidence strings dropped due to unresolved disease', previous_run, latest_run)
```

Reply:
This is great! Much neater.
Fixed in `26714ce`